src/errors/errors.ts

```diff
@@ -91,7 +91,7 @@ export class UnsupportedChainError extends APError {
   constructor(unsupportedChainId: string) {
     super(
       "UnsupportedChainError",
-      `Chain ID ${unsupportedChainId} not supported. The only supported networks are on: Juno, Polygon, Terra, Binance and Ethereum. Please switch to one of those and reload the page`
+      `Chain ID ${unsupportedChainId} not supported. The only supported networks are ${EXPECTED_NETWORK_TYPE} chains on: Juno, Polygon, Terra, Binance and Ethereum. Please switch to one of those and reload the page`
```

Comment:
```suggestion
      `The chain you are connected to (ID: ${unsupportedChainId}) is not supported. Supported networks are ${EXPECTED_NETWORK_TYPE} chains on: Juno, Polygon, Terra, Binance and Ethereum. Please switch to one of those and reload the page.`
```

Reply:
@SovereignAndrey we could add the supported chain IDs next to their respective chain names using [chainIds](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/constants/chainIds.ts#L4)? 🤔
It automatically detects the correct network (mainnet/testnet) of the supported chains. 

So instead of message reading:
```ts
`Valid networks are ${EXPECTED_NETWORK_TYPE} chains on: Juno, Polygon, Terra, Binance and Ethereum.`
```
it could read:
```ts
`Valid networks are ${EXPECTED_NETWORK_TYPE} chains on: Juno (${chainIds.juno}), Polygon (${chainIds.polygon}), Terra (${chainIds.terra}), Binance (${chainIds.binance}) and Ethereum (${chainIds.ethereum}).`
```

Reply:
That works nicely to tie this chain ID concept to a human readable name for normies. 👌

Reply:
```suggestion
      `The chain you are connected to (ID: ${unsupportedChainId}) is not supported. Valid networks are ${EXPECTED_NETWORK_TYPE} ones on: Juno (${chainIds.juno}), Polygon (${chainIds.polygon}), Terra (${chainIds.terra}), Binance (${chainIds.binance}) and Ethereum (${chainIds.ethereum}). Please switch to one of those and reload the page.`
```

@SovereignAndrey updated `Valid networks are ${EXPECTED_NETWORK_TYPE} CHAINS on` to `Valid networks are ${EXPECTED_NETWORK_TYPE} ONES on` (changed: "chains" -> "ones").